tests/e2e/tests/bookinfo/demo_test.go

```diff
@@ -400,6 +400,8 @@ func setTestConfig() error {
 	return nil
 }
 
+/** Disabled because mixer_test has a better DB test. Keeping this
```

Comment:
let's not disable it - it doesn't take long, it doesn't hurt to have 2 tests, and it was the only smoking gun of some auth issue